app/views/ppm_estimator/index.html.erb

```diff
@@ -0,0 +1,156 @@
+<%- @page_title = 'Personally Procured Move (PPM) Estimator' -%>
+
+<%= render partial: 'shared/contextual_header', locals: {page_title: @page_title, parent_pages: [{name: 'Tools & Resources', path: page_path('resources')}]} %>
+
+<div class="usa-section main-section">
+  <h1><%= @page_title %></h1>
+
+<h4>
+    For your upcoming move, get an estimate for how much the government will
+    pay <strong>you</strong> if you choose to "do it yourself."
+  </h4>
+
+  <p>
+    You can use this money to cover the cost of hiring packers and movers,
+    renting moving trucks, and so on. If you spend less to move than the
+    government will incentivize you to move yourself, you get to pocket the
+    difference!
+  </p>
+
+  <%= form_with url: ppm_estimator_path, class: 'ppm-form', id: 'ppm-estimate-form', method: 'get' do |form| %>
+    <fieldset>
+      <legend><div class="ppm-outline"></div>What is your rank?</legend>
```

Comment:
Is it possible to hang all of the styling (and `content`) off of `legend::before` or `legend::after` instead of having this empty `<div>`?

Reply:
yes, that worked well.